ruby/lib/trailer_vote/api/issue/create.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'trailer_vote/api/issue'
```

Comment:
Missing a require to `find`
```suggestion
require 'trailer_vote/api/issue'
require 'trailer_vote/api/issue/find'
```